pkg/rtc/mediatrack.go

```diff
@@ -28,6 +30,9 @@ type MediaTrack struct {
 	buffer      *buffer.Buffer
 
 	*MediaTrackReceiver
+	*MediaLossProxy
+
+	dynacastManager *DynacastManager
```

Comment:
These got moved out of `MediaTrackReceiver` and `MediaTrackSubscriptions` into here so that local media track is managed here and remote media track is managed in relay manager.

Reply:
great, I think this is cleaner!